pkg/kubelet/kubelet_node_status.go

```diff
@@ -273,6 +273,23 @@ func (kl *Kubelet) initialNode() (*v1.Node, error) {
 			glog.Infof("Adding node label from cloud provider: %s=%s", metav1.LabelInstanceType, instanceType)
 			node.ObjectMeta.Labels[metav1.LabelInstanceType] = instanceType
 		}
+
+		labels, err := instances.NodeLabels(kl.nodeName)
+		if err != nil {
+			return nil, err
+		}
+		if labels != nil {
+			prefix = "cloud.alpha.kubernetes.io/"
+			for name, value := range labels {
+				key := prefix + name
+				if len(key) > 63 {
+					glog.Warningf("Key: %s is too long, skipping", key)
```

Comment:
i am a little worried this could cause spam.  do clouds often have labels larger than this?  we have been hit by a lot of spam logs in production as of late and have been making an effort to reduce it.